lib/capybara/rack_test/driver.rb

```diff
@@ -50,6 +50,9 @@ def refresh
     browser.refresh
   end
 
+  def send_keys(*)
```

Comment:
If this can't do anything it should be an error to call it

Reply:
Thanks for clarifying @twalpole. Is declaring it still worthwhile? Is there a better (more Capybara-specific) alternative to `NoMethodError`?

Reply:
Yes - we should have a specific error mentioning that the rack test driver doesn't have a concept of active element -- it should not be a nomethoderror